### PR TITLE
Update rbac.md

### PR DESCRIPTION
**Pull request by AdamDang**
In the main text and paragraph title, "Tiller" and "tiller" are both used.
It's better use same format.

### CODE REVIEW

File: docs/rbac.md [hunk 1]
```diff
@@ -224,11 +224,11 @@ rolebinding "tiller-binding" created
 
 ## Helm and Role-based Access Control
 
-When running a helm client in a pod, in order for the helm client to talk to a tiller instance, it will need certain privileges to be granted. Specifically, the helm client will need to be able to create pods, forward ports and be able to list pods in the namespace where tiller is running (so it can find tiller).
+When running a helm client in a pod, in order for the helm client to talk to a Tiller instance, it will need certain privileges to be granted. Specifically, the helm client will need to be able to create pods, forward ports and be able to list pods in the namespace where Tiller is running (so it can find Tiller).
```
Comment (thomastaylor312) [inline docs/rbac.md]: While we are fixing capitalization, can you capitalize "Helm" where it occurs in this sentence?

File: docs/rbac.md [hunk 2]
```diff
@@ -224,11 +224,11 @@ rolebinding "tiller-binding" created
...
 
 ### Example: Deploy Helm in a namespace, talking to Tiller in another namespace
 
-In this example, we will assume tiller is running in a namespace called `tiller-world` and that the helm client is running in a namespace called `helm-world`. By default, tiller is running in the `kube-system` namespace.
+In this example, we will assume Tiller is running in a namespace called `tiller-world` and that the helm client is running in a namespace called `helm-world`. By default, Tiller is running in the `kube-system` namespace.
```
Comment (thomastaylor312) [inline docs/rbac.md]: Capitalize Helm here as well

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
@AdamDang Thanks for fixing this! We really appreciate it when people fix up docs. Just a few quick changes for capitalizing Helm as well while you are in here fixing this 🙂

**Comment by AdamDang** [comment]
@thomastaylor312 Thanks for your kind attention.  Capitalizing Helm is done.

**Review by thomastaylor312 — APPROVED** [review]
Thank you for the changes!

**Pull request merged**